Packages/com.unity.inputsystem/InputSystem/Devices/Pointer.cs

```diff
@@ -57,20 +56,16 @@ internal struct PointerState : IInputStateTypeInfo
         [InputControl(layout = "Analog", usage = "Pressure")]
         public float pressure;
 
-        [InputControl(layout = "Axis", usage = "Twist")]
-        public float twist;
-
-        [InputControl(layout = "Vector2", usage = "Tilt")]
-        public Vector2 tilt;
+        float twist;
+        Vector2 tilt;
```

Comment:
These can be removed.

Reply:
Nope, that will break il2cpp struct layout.

Reply:
That's fine. These are not used by native. The struct doesn't have to adhere to any particular layout.